docs/en_US/builtin_tuner.rst

```diff
@@ -14,6 +14,7 @@ Tuner receives metrics from `Trial` to evaluate the performance of a specific pa
     Random Search <Tuner/HyperoptTuner>
     Anneal <Tuner/HyperoptTuner>
     Naive Evolution <Tuner/EvolutionTuner>
+    Evolution Tuner <Tuner/EvolutionNasTuner>
```

Comment:
please put it in NAS doc as a tuner for classic NAS: https://nni.readthedocs.io/en/latest/NAS/Overview.html